### ISSUE TITLE
should every F2F agenda include CEPC in short?

### ISSUE THREAD

**Issue by TzviyaSiegman** [w3c/PWETF#55]
@jorydotcom has pointed out that JTC1 SC includes a slide they're using ahead of all their meetings/presentations about following a code of conduct. 

> The agenda item should be at the beginning of the agenda, should provide a link to the Codes of Conduct and should include this slide (projected) that emphasizes key messages concerning
> ethics and respect.

The required slide is basically a code of conduct in a bullet list. See https://www.iso.org/publication/PUB100397.html and https://basecamp.iec.ch/download/iec-code-of-conduct-for-delegates-and-experts/.
Should W3C require slides to include CEPC or a short version of CEPC?

**Comment by AdaRoseCannon** [comment]
I think so, we do so in the Immersive Web Working Group already. I think reminding people about it's presence is beneficial.

**Comment by jorydotcom** [comment]
+1!

**Comment by RachelComerford** [comment]
+1

**Comment by TzviyaSiegman** [comment]
Note that ietf includes similar in their "note well" document https://www.ietf.org/about/note-well/

**Comment by koalie** [comment]
+1

Note: for a few years now, there's been a mention in web pages for the W3C Advisory Committee Meetings and W3C TPACs, as well as a dedicated slide at least in the CEO slides.

For example: 
 
> Code of Conduct
> 
>  TPAC 2019 operates under CEPC.
> 
>  We all share responsibility for our work environment.
> 
>     Respect, professionalism, fairness, and sensitivity.
>     No harassment, no bullying.
>     No discrimination on any basis.
>     Constructive dialog. No insult.
> 
>  CEPC is developped in the PWE CG.

cf. @jeffjaffe's most recent (public) deck: slide 4 "[code of conduct](https://www.w3.org/2019/Talks/TPAC/ceo-tpac-update/#cepc)"

**Comment by nigelmegitt** [comment]
Of course, I agree with the issue that the CEPC should be included in every F2F agenda.

However, why are we restricting this to _F2F_ agendas, and not making it universal to all meetings, whether in person or remote? Surely it applies equally to both.
